Makefile

```
@@ -14,9 +14,19 @@ install: ## Install all dependencies
 	cd packages/periscope && poetry install
 	cd packages/oracle && poetry install
 
-dev: ## Install development dependencies
+dev: ## Install development dependencies and run full development cycle
```

checkmake (0.2.2)

[warning] Target body for "dev" exceeds allowed length of 5 (12).

(maxbodylength)